### PR TITLE
feat(indent): add indent-rainbowline

### PR DESCRIPTION
**Pull request by fuchami**
## 📑 Description

Add rainbow indents plugin.

https://github.com/TheGLander/indent-rainbowline.nvim

### CODE REVIEW

**Comment by Uzaaft** [comment]
https://github.com/AstroNvim/astrocommunity/issues/634

**Comment by Uzaaft** [comment]
Feel free to re-open this for v4 to Get it merged ASAP

**Comment by fuchami** [comment]
I plan to add this feature after v4 is released. thank you.

**Comment by Uzaaft** [comment]
R? @AstroNvim/astrocommunity-maintainers

File: lua/astrocommunity/indent/indent-rainbowline/init.lua [hunk 1]
```diff
@@ -0,0 +1,10 @@
+return {
+  {
+    "lukas-reineke/indent-blankline.nvim",
+    event = "User AstroFile",
+    opts = function(_, opts) return require("indent-rainbowline").make_opts(opts) end,
+    dependencies = {
+      "TheGLander/indent-rainbowline.nvim",
+    },
+  },
```
Comment (mehalter) [inline lua/astrocommunity/indent/indent-rainbowline/init.lua:9]:
```suggestion
  "lukas-reineke/indent-blankline.nvim",
  opts = function(_, opts)
    return require("indent-rainbowline").make_opts(opts, require("astrocore").plugin_opts "indent-rainbowline.nvim")
  end,
  dependencies = { "TheGLander/indent-rainbowline.nvim" },
```

**Review by mehalter — COMMENTED** [review]
Added a suggestion that would allow users to use the `opts` table for `indent-rainbowline` to configure the `make_opts` call

**Comment by Uzaaft** [comment]
LGTM now. :) Any last words @mehalter

**Review by mehalter — APPROVED** [review]
LGTM

**Pull request merged**